src/Microsoft.XmlSerializer.Generator/src/Sgen.cs

```diff
@@ -467,5 +475,77 @@ private static string GetTempAssemblyName(AssemblyName parent, string ns)
         {
             return parent.Name + ".XmlSerializers" + (ns == null || ns.Length == 0 ? "" : "." + ns.GetHashCode());
         }
+
+        private static void ParseReferences(string value, Dictionary<string, string> dictionary)
+        {
+            List<string> list = new List<string>();
+            if (value.Length > 0)
+            {
+                string[] entries = value.Split(new char[] { ';' });
```

Comment:
Can we use Split(char) here instead of allocating a single char array?
https://github.com/dotnet/coreclr/blob/master/src/mscorlib/shared/System/String.Manipulation.cs#L1156

Reply:
string.Split method seems not take a single char as input parameter.